docs/fidesops/docs/guides/complex_fields.md

```diff
@@ -267,8 +267,9 @@ arrays should be considered.
 1) If an array is the entry point into a node, we will search for corresponding matches across the entire array. You cannot specify a certain index.
 2) Everything is basically an "OR" query. Data returned from multiple array fields will be flattened before being passed into the next collection.
    1) For example, say Collection A returned values [1, 2, 3] and Collection B returned values [4, 5, 6].  Collection C has an array field that depends on both Collection A and Collection B. We search Collection C's array field to return any record that contains one of the values [1, 2, 3, 4, 5, 6] in the array.
-3. If an array field is an entry point to a node, only matching indices in that array are considered, both for access and erasures, as well as for subsequent queries on dependent collections where applicable.
-   1. For example, a query on Collection A only matched indices 0 and 1 in an array.  Only the data located at indices 0 and 1 are used to query data on dependent collection C.
+3. By default, if an array field is an entry point to a node, only matching indices in that array are considered, both for access and erasures, as well as for subsequent queries on dependent collections where applicable.
+   1. For example, a query on Collection A only matched indices 0 and 1 in an array.  Only the data located at indices 0 and 1 will be returned, and used to query data on dependent collection C.
+   2. This can be overridden by specifying `return_all_elements=true` on an entrypoint array field, in which case, the query will return the entire array and mask the entire array.
```

Comment:
This is a great description 👍 . Are we better off using yaml notation in the example we give, since that's consistent with the example datasets? ie. `return_all_elements: true`

Reply:
ah good point